graspologic/inference/latent_distribution_test.py

```diff
@@ -26,8 +26,25 @@
 
 _VALID_TESTS = ["cca", "dcorr", "hhg", "rv", "hsic", "mgc"]
 
-
-class LatentDistributionTest(BaseInference):
+ldt_result = namedtuple(
```

Comment:
name of the last thingee

Reply:
shouldn't there be a test that would break if something like this happens? i.e. if for the return types/ names?